Cargo.toml

```diff
@@ -17,23 +17,28 @@ edition = "2021"
 # Command-line
 clap = { version = "3", default-features = false, features = ["std", "cargo"] }
 # Server
-tokio = { version = "1", features = ["rt", "macros"]}
-hyper = { version = "0.14.10", features = ["http1", "server", "tcp"] }
+tokio = { version = "1", features = ["rt-multi-thread", "macros"] }
+tokio-util = { version = "0.7", features = ["io"] }
```

Comment:
Where do we use `tokio-util`?

Reply:
I previously use it with async-compression. It can be removed since we don't use async-compression anymore. Removed on 481b1fb